cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -156,9 +157,109 @@ def plot_histogram(self, ax: Optional[plt.Axes] = None, **plot_kwargs) -> plt.Ax
             fig.show(**plot_kwargs)
         return ax
 
+    @cached_method
+    def pauli_error(self) -> Dict[Tuple['cirq.GridQubit', 'cirq.GridQubit'], float]:
+        """Return the Pauli error of a qubit pair."""
+        return {
+            pair: noise_utils.decay_constant_to_pauli_error(
+                noise_utils.xeb_fidelity_to_decay_constant(self.xeb_fidelity(*pair), num_qubits=2),
+                num_qubits=2,
+            )
+            for pair in self.all_qubit_pairs
+        }
+
+
+@dataclass(frozen=True)
+class CombinedXEBRBResult:
```

Comment:
Maybe we can give this a more informative name and docstring. "Combined" makes it sound like it's telling you about both 1Q and 2Q fidelities, whereas the idea is really to isolate the 2Q fidelities.

Reply:
how about ?
```py3
@dataclass(frozen=True)
class InferredXEBRBResult:
    """Uses the results from XEB and RB to compute inferred errors."""
```